nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/Constants.java

```diff
@@ -65,4 +65,6 @@ private Constants() {
 
     public static final String TOPIC_TYPE = "Thread";
 
+    public static final String COLLECTION_TYPE = "Collection";
+
```

Comment:
perhaps i missed something, but i didn't find the usage of this new constant 
thx :)

Reply:
It is used here: https://github.com/nuxeo/nuxeo-jsf-ui/pull/35/commits/757631b765d2ad230d3e274d65a28c84f7b6ed67#diff-518602429846854c7809146474e41843R98

Reply:
yes thanks lot i completely forgot about the jsf, on master it's a separate repo ;)